packages/cli/src/commands/account.ts

```diff
@@ -143,7 +144,9 @@ const getLoginConfig = async (
 const createConfigFromLoginParameters = (loginParameters: string[]) => (
   async (credentialsType: ObjectType): Promise<InstanceElement> => {
     const configValues = Object.fromEntries(loginParameters.map(entryFromRawLoginParameter))
-    const missingLoginParameters = Object.keys(credentialsType.fields)
+    const requiredFields = Object.fromEntries(Object.entries(credentialsType.fields)
+      .filter(([_fieldName, field]) => field.annotations[CORE_ANNOTATIONS.REQUIRED] !== false))
```

Comment:
nit: we only really need the field names, so can do something like
```suggestion
    const requiredFields = Object.entries(credentialsType.fields)
      .filter(([_fieldName, field]) => field.annotations[CORE_ANNOTATIONS.REQUIRED] !== false)
      .map(([fieldName]) => fieldName)
```
or
```suggestion
    const requiredFields = Object.keys(credentialsType.fields)
      .filter(fieldName => credentialsType.fields[fieldName].annotations[CORE_ANNOTATIONS.REQUIRED] !== false))
```